src/elements/common/item/iconCellRenderer.js

```diff
@@ -12,15 +12,19 @@ import { TYPE_FOLDER, TYPE_FILE, TYPE_WEBLINK } from '../../../constants';
 
 import './IconCell.scss';
 
-export function getIcon(dimension: number, rowData: BoxItem) {
+export function getIcon(dimension: number, rowData: BoxItem, changeWeblinkHeight: boolean = false) {
     const { type, extension, has_collaborations, is_externally_owned }: BoxItem = rowData;
     switch (type) {
         case TYPE_FOLDER:
             return <FolderIcon dimension={dimension} isCollab={has_collaborations} isExternal={is_externally_owned} />;
         case TYPE_FILE:
             return <FileIcon dimension={dimension} extension={extension} />;
         case TYPE_WEBLINK:
-            return <BookmarkIcon width={dimension} />;
+            return changeWeblinkHeight ? (
```

Comment:
Is there a better way to do this?  Not sure why this was only changing `width` and not `height`, but I felt this approach was safer and wouldn't change behavior in other areas where we use `getIcon`.

Reply:
Looks like this is only used by ContentExplorer iconCellRenderer so it seems like it should be ok to use height and width